app/controllers/api/v1/api_controller.rb

```diff
@@ -4,14 +4,27 @@ class ApiController < ActionController::API
       before_action :auth
 
       def current_user
-        @current_user ||= User.find_by(token: token)
+        @current_user ||= if doorkeeper_token
+                            User.find(doorkeeper_token.resource_owner_id)
```

Comment:
This `.find` is going to raise an ActiveRecord::RecordNotFound in opposite to the `.find_by` that's gonna return `nil`, is that intentional?